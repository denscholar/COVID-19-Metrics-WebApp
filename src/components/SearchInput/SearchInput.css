.input {
  padding: 5px 20px;
  background-color: #f43659;
  position: relative;
  width: 100%;
  display: flex;
  justify-content: center;
  height: 3.5rem;
  align-items: center;
  align-content: center;
}

nav input {
  width: 40%;
  border-radius: 4px;
  outline: none;
  border: none;
  padding: 5px 15px;
}
