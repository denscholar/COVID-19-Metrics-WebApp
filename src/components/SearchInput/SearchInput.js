import './SearchInput.css';

const SearchInput = ({ onChange }) => (
  <nav className="input">
    <input type="search" placeholder="Search country cases" onChange={onChange} />
  </nav>
);

export default SearchInput;
