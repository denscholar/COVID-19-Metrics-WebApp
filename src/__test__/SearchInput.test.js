import React from 'react';
import { create } from 'react-test-renderer';
import { cleanup } from '@testing-library/react';
import { Provider } from 'react-redux';
import store from '../redux/configureStore';
import SearchInput from '../components/SearchInput/SearchInput';

afterEach(() => cleanup());

it('App Renders Correctly', () => {
  const tree = create(
    <Provider store={store}>
      <SearchInput />
    </Provider>,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
