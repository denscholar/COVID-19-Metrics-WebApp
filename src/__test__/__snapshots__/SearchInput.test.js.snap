// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Renders Correctly 1`] = `
<nav
  className="input"
>
  <input
    placeholder="Search country cases"
    type="search"
  />
</nav>
`;
